Neos.ContentRepository.LegacyNodeMigration/Classes/Command/CrCommandController.php

```diff
@@ -164,6 +164,7 @@ private function adjustDataBaseConnection(Connection $connection): Connection
         $connectionParams['dbname'] = $this->output->ask(sprintf('DB name? [%s] ',$connectionParams['dbname'] ?? ''), $connectionParams['dbname'] ?? null);
         $connectionParams['user'] = $this->output->ask(sprintf('DB user? [%s] ',$connectionParams['user'] ?? ''), $connectionParams['user'] ?? null);
         $connectionParams['password'] = $this->output->askHiddenResponse(sprintf('DB password? [%s]', str_repeat('*', strlen($connectionParams['password'] ?? '')))) ?? $connectionParams['password'];
```

Comment:
Well if I follow the codebase it says it does have that... The Schema is described in the DBAL DriverManager class at the top and it has password 🤷 I guess phpstan ignore? 

That said `$connection->getParams();` (line 159) is internal so this whole thing is on shaky foundations...